src/Hazelcast.Net/Networking/SocketConnectionBase.cs

```diff
@@ -231,7 +231,6 @@ private async ValueTask ShutdownInternal(Task task)
             _streamReadCancellationTokenSource.Cancel();
 
             // ensure everything is down by awaiting the other task
-            // FIXME why could this be null ?!
```

Comment:
So have you found an answer? :)

Reply:
No :) I don't think it can ben null & I don't remember why this warning is here, so I remove it.